### ISSUE TITLE
Visible border issue around the generated images

### ISSUE THREAD

**Issue by OAHLSTM** [AlbertoAncilotto/XiNet_style_transfer#1]
Hello,

I have been trying to reproduce the training of the model using the exact same scripts provided in your repo in the Jupyter Notebook train_notebook.ipynb by using the whole COCO2017 dataset as content images and the muse_picasso as the style image. I have been training the model for 120 epochs. I am facing an issue on the generated images during the training and a significant part of the content image is lost in the top, bottom, right and left borders of the image, as can be seen in the following figure.

<img width="1024" height="512" alt="Image" src="https://github.com/user-attachments/assets/72a120fd-bb3c-4510-934b-bca734c9c3bd" />

I have been trying to investigate this issue since few weeks, and by fine tuning the weights of each of the loss functions, I have been reaching a better performance with the variation weight set to 1e-4 instead of 5e-3. I am noticing that the area in the borders is reduced but not completely disappearing, as shown in the following figure. 

<img width="1024" height="512" alt="Image" src="https://github.com/user-attachments/assets/775ce83d-28e8-40a5-8f51-0c1af5c14eb1" />

I was wondering if this problem sounds familiar to you or if you have faced anything similar. What do you think could be the root cause of this issue ? 
I would appreciate your help on this issue.
Thank you,

**Comment by OAHLSTM** [comment]
Hello @AlbertoAncilotto ,
Do you have any idea about the root cause of this issue ?
Thank you,
